Code/GraphMol/DistGeomHelpers/Embedder.h

```diff
@@ -381,14 +384,14 @@ inline void EmbedMultipleConfs(
     bool useBasicKnowledge = false, bool verbose = false,
     double basinThresh = 5.0, bool onlyHeavyAtomsForRMS = false,
     unsigned int ETversion = 1, bool useSmallRingTorsions = false,
-    bool useMacrocycleTorsions = false, bool useMacrocycle14config = false) {
+    bool useMacrocycleTorsions = false, double boundsMatForceScaling = 1.0, bool useMacrocycle14config = false) {
```

Comment:
you can't introduce new parameters in such a way that they break the current API.
Please add the new parameter at the end of the argument list